FirebaseInstallations/Source/Library/InstallationsIDController/FIRInstallationsIDController.m

```diff
@@ -75,8 +75,12 @@ - (instancetype)initWithGoogleAppID:(NSString *)appID
   FIRSecureStorage *secureStorage = [[FIRSecureStorage alloc] init];
   FIRInstallationsStore *installationsStore =
       [[FIRInstallationsStore alloc] initWithSecureStorage:secureStorage accessGroup:accessGroup];
+
+  // Use `GCMSenderID` as `projectID` is not available.
```

Comment:
Please change to:
"User `GCMSenderID` as project identifier, if `projectID` is not set"
(or "is not available")

Reply:
Good catch!